.goreleaser.yml

```diff
@@ -11,8 +11,8 @@ archives:
   - files:
       - src: LICENSE
         dst: LICENSE.txt
-    formats: [ 'zip' ]
-    name_template: "az{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
+    formats: [zip]
+    name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
```

Comment:
@DariuszPorowski should we leave the "az" prefix to the executable as agreed earlier? If not then we will need to change the docs